fairseq/tasks/hubert_pretraining.py

```diff
@@ -128,24 +132,21 @@ def source_dictionary(self) -> Optional[Dictionary]:
 
     @property
     def target_dictionary(self) -> Optional[Dictionary]:
-        return self._target_dictionary
+        return self.state.target_dictionary
 
     @property
     def dictionaries(self) -> List[Dictionary]:
-        return [self._dictionaries[l] for l in self.cfg.labels]
+        return self.state.dictionaries
```

Comment:
I think this needs to be: `return self.state.dictionaries if not self.fine_tuning else [self.state.target_dictionary]`

Reply:
do we need that? I think `dictionaries` is only used for pre-training; whereas `target_dictionary` is used for fine-tuning